apps/services/auth-public-api/src/environments/environment.ts

```diff
@@ -26,6 +26,15 @@ const devConfig = {
       clientId: 'IS-DEV/GOV/10000/island-is-client',
       memberClass: XRoadMemberClass.GovernmentInstitution,
     },
+    authMiddlewareOptions: {
+      forwardUserInfo: false,
+      tokenExchangeOptions: {
+        issuer: 'https://identity-server.dev01.devland.is',
+        clientId: '@island.is/clients/national-registry',
+        clientSecret: process.env.NATIONAL_REGISTRY_IDS_CLIENT_SECRET,
+        scope: '@skra.is/individuals api_resource.scope', // TODO: remove api_resource.scope
```

Comment:
Why is api_resource.scope needed? What's the plan? Just curious.

Reply:
This is what Þjóðskrá used in the beginning and changing it has not been a priority for them. Should be removed soon.